spectrum-offchain-lm/src/data/order.rs

```diff
@@ -351,6 +390,8 @@ pub struct Redeem {
     pub redeemer_prop: ErgoTree,
     pub bundle_key: TypedAssetAmount<BundleKey>,
     pub expected_lq: TypedAssetAmount<Lq>,
+    pub miner_prop_bytes: Vec<u8>,
```

Comment:
MinerProposition is constant, so no need to parse it.

Reply:
So we just need to check against this: https://github.com/ergoplatform/sigma-rust/blob/46c70f58d5f552b08c20af4fc8af91386e488aae/ergo-lib/src/wallet/miner_fee.rs#L8-L9
?

Reply:
Yes

Reply:
Thanks, all fixed.